Source/Core/DolphinQt/NetPlay/NetPlayDialog.cpp

```diff
@@ -910,6 +941,22 @@ std::shared_ptr<const UICommon::GameFile> NetPlayDialog::FindGameFile(const std:
   return nullptr;
 }
 
+void NetPlayDialog::SaveSettings()
+{
+  Config::SetBaseOrCurrent(Config::NETPLAY_HOST_BUFFER_SIZE,
```

Comment:
Use `SetBase` instead, as we want these to persist outside of any currently active game session.